### PR TITLE
data/aws: try to make startup faster by eliminating dependencies in DNS record creation

### PR DESCRIPTION
**Pull request by eparis**
We need the API DNS records in order for ignition to run on the masters. But today we need the master's IP addresses to create the etcd DNS records. This PR splits the creation of the API DNS record (which doesn't need to know the master IPs) from the etcd records. In this way the api dns records are created much sooner.

### CODE REVIEW

**Comment by eparis** [comment]
This includes the PR from https://github.com/openshift/installer/pull/1308

**Comment by abhinavdahiya** [comment]
nit: what about ?
```console
/route53
  /api
  /etcd
```

**Comment by eparis** [comment]
You're asking mostly from a file on disk layout, not from a main.tf flow? I would still need to source both `./route53/api` and `./route53/etcd` in main.tf, I think. If I just source a new module in `./route53` it wouldn't break the route53->master dep, right?

**Comment by eparis** [comment]
I just read https://www.terraform.io/docs/modules/usage.html#outputs and might just be wrong...

File: data/data/aws/main.tf [hunk 1]
```diff
@@ -52,8 +52,8 @@ module "iam" {
   tags = "${local.tags}"
 }
 
-module "dns" {
-  source = "./route53"
+module "dns-api" {
+  source = "./route53/api"
```
Comment (wking) [inline data/data/aws/main.tf]: I think this is an opportunity to resolve the `dns` vs. `route53` naming difference.  I don't care which we standardize on, but I'd like to have the module name match the source path (e.g. `route53-api` and `./route53/api` or `dns-api` and `./dns/api`).

Comment (abhinavdahiya) [reply data/data/aws/main.tf]: IMO the current naming seems fine to me. @wking can you expand on why you think `but I'd like to have the module name match the source path` is something we should strive for.

Comment (wking) [reply data/data/aws/main.tf]: I just don't see a point to having two names for the same thing.

**Comment by abhinavdahiya** [comment]
### testing locally without this patch
bootstrap node:
```console
$ journalctl --list-boots
 0 65c6a6c642d34c659ffa85d5f5c09985 Tue 2019-02-26 18:47:36 UTC—Tue 2019-02-26 18:51:55 UTC
```
[mcs + etcd-signer]
```console
Feb 26 18:48:54 ip-10-0-6-72 bootkube.sh[4501]: Starting etcd certificate signer...
Feb 26 18:48:56 ip-10-0-6-72 bootkube.sh[4501]: e531fbfd7221944dc0aef001fe74de0a0613908b8b8a708cfa0a7202db6f015b
Feb 26 18:48:56 ip-10-0-6-72 bootkube.sh[4501]: Waiting for etcd cluster...
Feb 26 18:53:59 ip-10-0-6-72 bootkube.sh[4501]: https://etcd-2.adahiya-1.devcluster.openshift.com:2379 is healthy: successfully committed proposal: took = 2.439788ms
Feb 26 18:54:32 ip-10-0-6-72 bootkube.sh[4501]: https://etcd-1.adahiya-1.devcluster.openshift.com:2379 is healthy: successfully committed proposal: took = 3.589929ms
Feb 26 18:55:28 ip-10-0-6-72 bootkube.sh[4501]: https://etcd-0.adahiya-1.devcluster.openshift.com:2379 is healthy: successfully committed proposal: took = 2.813594ms
Feb 26 18:55:29 ip-10-0-6-72 bootkube.sh[4501]: etcd cluster up. Killing etcd certificate signer...
```

master-1 node:

```console
$ journalctl --list-boots
-1 0d3e50943a424e0ea4f1ff7be8e37d6e Tue 2019-02-26 18:47:48 UTC—Tue 2019-02-26 18:51:57 UTC
 0 8d395b350465463d9e25ad6bf1ce8667 Tue 2019-02-26 18:52:20 UTC—Tue 2019-02-26 18:56:01 UTC

```
[kubelet.service]
```console
Feb 26 18:52:31 localhost.localdomain systemd[1]: Starting Kubernetes Kubelet...
Feb 26 18:52:32 ip-10-0-148-78 hyperkube[4375]: Flag --rotate-certificates has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Feb 26 18:52:32 ip-10-0-148-78 hyperkube[4375]: Flag --allow-privileged has been deprecated, will be removed in a future version
Feb 26 18:52:32 ip-10-0-148-78 hyperkube[4375]: Flag --minimum-container-ttl-duration has been deprecated, Use --eviction-hard or --eviction-soft instead. Will be removed in a future version.
Feb 26 18:52:32 ip-10-0-148-78 hyperkube[4375]: Flag --client-ca-file has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
```
[ignition]
```console
$ journalctl -t ignition --no-pager
Feb 26 18:47:49 localhost ignition[486]: Ignition 0.30.0
Feb 26 18:47:49 localhost ignition[486]: reading system config file "/usr/lib/ignition/base.ign"
Feb 26 18:47:49 localhost ignition[486]: no config at "/usr/lib/ignition/base.ign"
Feb 26 18:47:49 localhost ignition[486]: parsed url from cmdline: ""
Feb 26 18:47:49 localhost ignition[486]: no config URL provided
Feb 26 18:47:49 localhost ignition[486]: reading system config file "/usr/lib/ignition/user.ign"
Feb 26 18:47:49 localhost ignition[486]: no config at "/usr/lib/ignition/user.ign"
Feb 26 18:47:49 localhost ignition[486]: GET http://169.254.169.254/2009-04-04/user-data: attempt #1
Feb 26 18:47:49 localhost ignition[486]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:47:49 localhost ignition[486]: GET http://169.254.169.254/2009-04-04/user-data: attempt #2
Feb 26 18:47:49 localhost ignition[486]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:47:50 localhost ignition[486]: GET http://169.254.169.254/2009-04-04/user-data: attempt #3
Feb 26 18:47:50 localhost ignition[486]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:47:51 localhost ignition[486]: GET http://169.254.169.254/2009-04-04/user-data: attempt #4
Feb 26 18:47:51 localhost ignition[486]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:47:52 localhost ignition[486]: GET http://169.254.169.254/2009-04-04/user-data: attempt #5
Feb 26 18:47:52 localhost ignition[486]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:47:55 ip-10-0-148-78.ec2.internal ignition[486]: GET http://169.254.169.254/2009-04-04/user-data: attempt #6
Feb 26 18:47:55 ip-10-0-148-78.ec2.internal ignition[486]: GET result: OK
Feb 26 18:47:55 ip-10-0-148-78.ec2.internal ignition[486]: parsing config: {"ignition":{"config":{"append":[{"source":"https://api.adahiya-1.devcluster.openshift.com:22623/config/master","verification":{}}]},"security":{"tls":{"certificateAuthorities":[{"source":"data:text/plain;charset=utf-8;base64,LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURFRENDQWZpZ0F3SUJBZ0lJWTdKVU16WnJrRnN3RFFZSktvWklodmNOQVFFTEJRQXdKakVTTUJBR0ExVUUKQ3hNSmIzQmxibk5vYVdaME1SQXdEZ1lEVlFRREV3ZHliMjkwTFdOaE1CNFhEVEU1TURJeU5qRTRORFUxTVZvWApEVEk1TURJeU16RTRORFUxTVZvd0pqRVNNQkFHQTFVRUN4TUpiM0JsYm5Ob2FXWjBNUkF3RGdZRFZRUURFd2R5CmIyOTBMV05oTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUF6dzhhU2JIejR6bTEKa1RIWmt4NU16QXE5eWJVTDhtZXR6ZzB4emJEdm1aWWNVZ3dBSXhjdm5nSUNIY3dYNzc3N1hKbnVXUlJsN3RiUApNSkdFMjlMdzFubmtsRC9qVnBjeGh3QmNncHR0aVdCMm5jOEE3YWxBRVpkak5HSlpraGJ3R2lLZFU4MkN4L3dlCkJXWWpnUC9Sb2ZGMC85SytIQ01Pb21iNm9uLytnMjJFQzBqaVdCUXVKNGRBRjhOT1Z4ZnlRaXVLdE1XRmpid2UKdzNpdGhSSm1PN3VTTlhER2JLR0YvV2hIV0k2bUxQV25VRURyVzJMcmZ3ZXNucVUxdzV0NHYwcXl1SDdBVEZlSgpFRU90VlJkS2llYy94MUc4SDYyTVJXdVBxaUxxWGVTcXNiQ0lBam9qOE9FSnZiakduc0FVOUVRb0gyQk04NFpFCmJoQmlTOUJNRHdJREFRQUJvMEl3UURBT0JnTlZIUThCQWY4RUJBTUNBcVF3RHdZRFZSMFRBUUgvQkFVd0F3RUIKL3pBZEJnTlZIUTRFRmdRVTd6by92cjNSSG42czlya2wxbkg0a1BUVWNmSXdEUVlKS29aSWh2Y05BUUVMQlFBRApnZ0VCQUIwU2F0Um1uZFVRa01XVkRnMzJISnJSOHZ4UTJpYjFBMW1STndoUkJqbjdoTTNXbnlhUHUwL09GcER2ClVvZFNwM0dqTlNvaG4yNUpMNDI5VWVBeXp6QVFuVEV1Y2ZKUDVDZzYxVHVCVlplK04wTjdVZDlGSFZFdFNLY2gKVlI5VXdCMTVXbEwxSU8vNnRITFo3d0RGOVU5bnR0ek1Hc1RBb3Rad2k5UUxtcEpDdnZzUy9yQzZ0clpsVm9raApqeWhIREhZWEhqencyT25VenV1QVlabjZTNWJmSjU0RkN6ZFdveHhGcFphUFUvZmgyMnJ2d0xIU1VIUHRna1ZuCjJyM0hwUHhOdHdDOW5wSFJXNjY3L0gwNlJWRG5MdktUOGQzMWN4cXduMlVPZWpwQjlOUUJtVXZzaGx0Smpwc2kKQURsVWxOT2JTYnFlRkxERFFoYnlFMm5RTGtRPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==","verification":{}}]}},"timeouts":{},"version":"2.2.0"},"networkd":{},"passwd":{},"storage":{},"systemd":{}}
Feb 26 18:47:55 ip-10-0-148-78.ec2.internal ignition[486]: Adding "root-ca" to list of CAs
Feb 26 18:47:55 ip-10-0-148-78.ec2.internal ignition[486]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #1
Feb 26 18:47:55 ip-10-0-148-78.ec2.internal ignition[486]: GET error: Get https://api.adahiya-1.devcluster.openshift.com:22623/config/master: dial tcp: lookup api.adahiya-1.devcluster.openshift.com on 10.0.0.2:53: no such host
Feb 26 18:47:56 ip-10-0-148-78.ec2.internal ignition[486]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #2
...
Feb 26 18:49:52 ip-10-0-148-78.ec2.internal ignition[486]: GET error: Get https://api.adahiya-1.devcluster.openshift.com:22623/config/master: dial tcp: lookup api.adahiya-1.devcluster.openshift.com on 10.0.0.2:53: no such host
Feb 26 18:49:57 ip-10-0-148-78.ec2.internal ignition[486]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #29
Feb 26 18:50:27 ip-10-0-148-78.ec2.internal ignition[486]: GET error: Get https://api.adahiya-1.devcluster.openshift.com:22623/config/master: dial tcp 10.0.168.193:22623: i/o timeout
Feb 26 18:50:32 ip-10-0-148-78.ec2.internal ignition[486]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #30
Feb 26 18:50:47 ip-10-0-148-78.ec2.internal ignition[486]: GET result: OK

```

rough timeline:
```
bootstrap node boots [18:47] ->
master-1 node boots [18:47] ->
mcs starts on bootstrap node [18:49] ->
master-1 starts kubelet (ie. setup by ignition) [18:52] ->
master-1 etcd boots [18:54] ->
bootstrap node starts control plane [18:55]
```

### testing locally with this patch

bootstrap node:
```console
$ journalctl --list-boots
 0 2b77ab0beb8543de976a8876fa7d59e9 Tue 2019-02-26 18:24:26 UTC—Tue 2019-02-26 18:31:54 UTC
```
[mcs + etcd-signer]
```console
Feb 26 18:25:41 ip-10-0-8-215 bootkube.sh[4502]: Starting etcd certificate signer...
Feb 26 18:25:47 ip-10-0-8-215 bootkube.sh[4502]: 9b45db7373deaf8e9998259cfb1f46c07ada7071b9d48449be87c840bf034075
Feb 26 18:25:47 ip-10-0-8-215 bootkube.sh[4502]: Waiting for etcd cluster...
Feb 26 18:30:20 ip-10-0-8-215 bootkube.sh[4502]: https://etcd-2.adahiya-1.devcluster.openshift.com:2379 is healthy: successfully committed pro
Feb 26 18:30:30 ip-10-0-8-215 bootkube.sh[4502]: https://etcd-1.adahiya-1.devcluster.openshift.com:2379 is healthy: successfully committed pro
Feb 26 18:30:31 ip-10-0-8-215 bootkube.sh[4502]: https://etcd-0.adahiya-1.devcluster.openshift.com:2379 is healthy: successfully committed pro
Feb 26 18:30:31 ip-10-0-8-215 bootkube.sh[4502]: etcd cluster up. Killing etcd certificate signer...
```

master-1 node:

```console
$ journalctl --list-boots
-1 71046a2a59bd4d55a8f1ebce81dc7aec Tue 2019-02-26 18:23:57 UTC—Tue 2019-02-26 18:28:52 UTC
 0 a7c27902f13a4cb4b46e21ee5f472ca9 Tue 2019-02-26 18:29:14 UTC—Tue 2019-02-26 18:32:54 UTC
```
[kubelet.service]
```console
Feb 26 18:29:25 localhost.localdomain systemd[1]: Starting Kubernetes Kubelet...
Feb 26 18:29:26 ip-10-0-149-186 hyperkube[4364]: Flag --rotate-certificates has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Feb 26 18:29:26 ip-10-0-149-186 hyperkube[4364]: Flag --allow-privileged has been deprecated, will be removed in a future version
Feb 26 18:29:26 ip-10-0-149-186 hyperkube[4364]: Flag --minimum-container-ttl-duration has been deprecated, Use --eviction-hard or --eviction-soft instead. Will be removed in a future version.
Feb 26 18:29:26 ip-10-0-149-186 hyperkube[4364]: Flag --client-ca-file has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
```
[ignition]
```console
$ journalctl -t ignition --no-pager
Feb 26 18:23:58 localhost ignition[489]: Ignition 0.30.0
Feb 26 18:23:58 localhost ignition[489]: reading system config file "/usr/lib/ignition/base.ign"
Feb 26 18:23:58 localhost ignition[489]: no config at "/usr/lib/ignition/base.ign"
Feb 26 18:23:58 localhost ignition[489]: parsed url from cmdline: ""
Feb 26 18:23:58 localhost ignition[489]: no config URL provided
Feb 26 18:23:58 localhost ignition[489]: reading system config file "/usr/lib/ignition/user.ign"
Feb 26 18:23:58 localhost ignition[489]: no config at "/usr/lib/ignition/user.ign"
Feb 26 18:23:58 localhost ignition[489]: GET http://169.254.169.254/2009-04-04/user-data: attempt #1
Feb 26 18:23:58 localhost ignition[489]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:23:58 localhost ignition[489]: GET http://169.254.169.254/2009-04-04/user-data: attempt #2
Feb 26 18:23:58 localhost ignition[489]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:23:59 localhost ignition[489]: GET http://169.254.169.254/2009-04-04/user-data: attempt #3
Feb 26 18:23:59 localhost ignition[489]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:24:00 localhost ignition[489]: GET http://169.254.169.254/2009-04-04/user-data: attempt #4
Feb 26 18:24:00 localhost ignition[489]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:24:01 localhost ignition[489]: GET http://169.254.169.254/2009-04-04/user-data: attempt #5
Feb 26 18:24:01 localhost ignition[489]: GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
Feb 26 18:24:04 ip-10-0-149-186.ec2.internal ignition[489]: GET http://169.254.169.254/2009-04-04/user-data: attempt #6
Feb 26 18:24:04 ip-10-0-149-186.ec2.internal ignition[489]: GET result: OK
Feb 26 18:24:04 ip-10-0-149-186.ec2.internal ignition[489]: parsing config: {"ignition":{"config":{"append":[{"source":"https://api.adahiya-1.devcluster.openshift.com:22623/config/master","verification":{}}]},"security":{"tls":{"certificateAuthorities":[{"source":"data:text/plain;charset=utf-8;base64,LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURFRENDQWZpZ0F3SUJBZ0lJY29WaGdjamVkVnd3RFFZSktvWklodmNOQVFFTEJRQXdKakVTTUJBR0ExVUUKQ3hNSmIzQmxibk5vYVdaME1SQXdEZ1lEVlFRREV3ZHliMjkwTFdOaE1CNFhEVEU1TURJeU5qRTRNakl5TVZvWApEVEk1TURJeU16RTRNakl5TVZvd0pqRVNNQkFHQTFVRUN4TUpiM0JsYm5Ob2FXWjBNUkF3RGdZRFZRUURFd2R5CmIyOTBMV05oTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUF6NlR3SmVNNFk5MVoKQU4rWFJGenFIZnB0azViemVNYUl2cDBzZUhGR05vbGIzYS9OZUdWa0hDS050NzBzZEx4ZFFvWnc0MDY0RUtXUgp4eDhIdktoeTJMaDVKRDM4cGZxQ0k4aGhLUVlLcHpxZUtGMTAzZVgyMVByK0d6N0JpaWUrWEwzSUp1RUpycURaCmZBSnVnRWVzMWZ6UENFc25vN2YwVE1xOElpK1VibUVsYVVWcVVNNjlPZkNScHRZMFRXTURKRE5aRS9XdnliZ08KTUhleTQ0d1RzVDFEcDdGSEN3Yjk2M2hhOUNYeXRTU1M3K2VRZGQxVFc2STRxeWRpMTdsOHRpUVh2bi9EOWllYQptY3A4YW1BcGRxRWtzWVlXR0s0eHhLV21YeXRkNFlQeS9ucURyaDVvMENzZy8wUDkxOTk0ZEdOb2tFcm5kM0tjCkk5UXg2NVBlVndJREFRQUJvMEl3UURBT0JnTlZIUThCQWY4RUJBTUNBcVF3RHdZRFZSMFRBUUgvQkFVd0F3RUIKL3pBZEJnTlZIUTRFRmdRVWU2dWdFV0NIMzdkRXJ5c0cxdW9na05sbE5MQXdEUVlKS29aSWh2Y05BUUVMQlFBRApnZ0VCQUFoamxpM2NCSTViTEloZURhQkFzdVU4NWErMTgvbHpSR3JIemV6S3F0dEwzb2dlVDFVY2dkTURLVUZtCm5LblA5dktCelE0Y0ZBeS9yd2hiUDJJaWt5dERUVmpsN3RtMEh6VFdsUnpDSktVVEtLRU8wa1JwUE0xSm1BeEgKNERPclRXTVdwS0FNbEN6VWtyYi92MjE1VTBpR01neWVmeldPUWE3OStRTVdJSXRMdW8vNW9YRmU2cFlMTXgwNApYS2liK0REcXd5VllERTJoKzFRREc1UmcvK3NUVW9HOVFNT1hsMW14SWFhOGdpanVaYzB5V3hSM2xrV2VzR0NxCnNqMnRUcTd6YXFhKzVpRjdrWUg4OTlWbFJhV3JzNktham5yaUI1S1VUaWlRWWZReWw0L2tObit3eE9RVTlHUlMKTXBTd1Z3YVZ4TjFrSHQ1Z05iMzZZWWFQV09RPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==","verification":{}}]}},"timeouts":{},"version":"2.2.0"},"networkd":{},"passwd":{},"storage":{},"systemd":{}}
Feb 26 18:24:04 ip-10-0-149-186.ec2.internal ignition[489]: Adding "root-ca" to list of CAs
Feb 26 18:24:04 ip-10-0-149-186.ec2.internal ignition[489]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #1
Feb 26 18:24:04 ip-10-0-149-186.ec2.internal ignition[489]: GET error: Get https://api.adahiya-1.devcluster.openshift.com:22623/config/master: dial tcp: lookup api.adahiya-1.devcluster.openshift.com on 10.0.0.2:53: no such host
Feb 26 18:24:05 ip-10-0-149-186.ec2.internal ignition[489]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #2
...
Feb 26 18:27:01 ip-10-0-149-186.ec2.internal ignition[489]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #40
Feb 26 18:27:01 ip-10-0-149-186.ec2.internal ignition[489]: GET error: Get https://api.adahiya-1.devcluster.openshift.com:22623/config/master: dial tcp: lookup api.adahiya-1.devcluster.openshift.com on 10.0.0.2:53: no such host
Feb 26 18:27:06 ip-10-0-149-186.ec2.internal ignition[489]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #41
Feb 26 18:27:36 ip-10-0-149-186.ec2.internal ignition[489]: GET error: Get https://api.adahiya-1.devcluster.openshift.com:22623/config/master: dial tcp 10.0.195.214:22623: i/o timeout
Feb 26 18:27:41 ip-10-0-149-186.ec2.internal ignition[489]: GET https://api.adahiya-1.devcluster.openshift.com:22623/config/master: attempt #42
Feb 26 18:27:46 ip-10-0-149-186.ec2.internal ignition[489]: GET result: OK
```

rough timeline:
```
bootstrap node boots [18:24] ->
master-1 node boots [18:23] ->
mcs starts on bootstrap node [18:25] ->
master-1 starts kubelet (ie. setup by ignition) [18:29] ->
master-1 etcd boots [18:29] ->
bootstrap node starts control plane [18:30]
```

in both the cases the igntion is wait on `no route to host` for ~3-4 minutes. So I don't think this change is helping us... ?

**Comment by eparis** [comment]
closing. this does nothing of value what so ever.

**Pull request closed without merging**